packages/client/src/components/LeaderboardPage/types.ts

```diff
@@ -0,0 +1,7 @@
+export type leaderDataType = {
```

Comment:
Типы называют с большой буквы. 
+ предлагаю в названии типа не постфикс Type добавлять, а префикс T (так короче название будет)
т.е. 
```suggestion
export type TLeaderData = {
```